app/src/main/res/layout/view_location_list_preference.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android" android:orientation="vertical" android:layout_width="match_parent"
+  android:layout_height="match_parent">
+  <EditText
+    android:id="@+id/filter"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:hint="@string/filter_gdg_by_name"/>
```

Comment:
needs to add padding to align with list items that use layout `android.R.layout.simple_list_item_single_choice`